include/bluetooth/bluetooth.h

```diff
@@ -1153,6 +1176,34 @@ int bt_le_per_adv_sync_delete(struct bt_le_per_adv_sync *per_adv_sync);
  */
 void bt_le_per_adv_sync_cb_register(struct bt_le_per_adv_sync_cb *cb);
 
+/**
+ * @brief Enables receiving periodic advertising reports for a sync.
+ *
+ * If the sync is already receiving the reports, -EALREADY is returned.
+ *
+ * @param per_adv_sync The periodic advertising sync object.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int bt_le_per_adv_sync_recv_enable(struct bt_le_per_adv_sync *per_adv_sync)
```

Comment:
I guess you didn't mean to put the implementation here.

Reply:
Nope :D